### PR TITLE
Add new story ZZB under Chinese folder

### PR DESCRIPTION
**Pull request by xuanaux**
Add new story ZZB under Chinese folder

### CODE REVIEW

**Comment by cbuckey-uda** [comment]
@casey-collab, you there?

File: Chinese/ZZB/zzb.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+至尊宝：谁说我斗鸡眼？我只是把视力集中在一点，以改变我以往对事物的看法！
```
Comment (casey-collab) [inline Chinese/ZZB/zzb.md]: It looks like there is no link to this new file.  Did you forget to commit that change?  Make sure you create (and commit) an option for each new "page" you create.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by cbuckey-uda** [comment]
Sorry, Casey has trouble when file names have non-ascii characters. This looks good to me! :footprints:

**Pull request merged**